### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by TariqAlQadi**
Add Categories, landing page and a small temporary bugfix for iframe sizing

### CODE REVIEW

File: components/Navigation/index.js [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
-        />
-      </Link>
-    </StyledNavigation>
+    <>
+      {pathname === "/" ? (
+        <Link href="/tutorials"></Link>
+      ) : (
+        <StyledNavigation>
+          <Link href="/tutorials">
+            <SVGIcon variant="home" width="40px" />
```
Comment (RobinWitt) [inline components/Navigation/index.js:16]: Your Home Icon doesn't toggle. When I click on the favorite page, both icons are highlighted.

File: pages/tutorials/index.js [hunk 2]
```diff
@@ -1,14 +1,70 @@
...
 
 export default function HomePage() {
-  const [list] = useAtom(beginnerList);
+  const [list] = useAtom(allTutorials);
+
+  //show list states
+  const [showBeginner, setShowBeginner] = useState(false);
+  const [showIntermediate, setShowIntermediate] = useState(false);
+  const [showAdvanced, setShowAdvanced] = useState(false);
+  const [showCoin, setShowCoin] = useState(false);
```
Comment (RobinWitt) [inline pages/tutorials/index.js:13]: I just had an idea for another solution, let's talk about it on Zoom :D

**Review by RobinWitt — APPROVED** [review]
Looks good to me!

File: pages/tutorials/index.js (hunk 2 above)
Comment (TariqAlQadi) [reply pages/tutorials/index.js:13]: You are right .. I am going to change the code!

**Pull request merged**